tasks/winbuildscripts/dobuild.bat

```diff
@@ -11,6 +11,7 @@ if NOT DEFINED GO_VERSION_CHECK set GO_VERSION_CHECK=%~4
 
 set OMNIBUS_BUILD=omnibus.build
 set OMNIBUS_ARGS=--python-runtimes "%PY_RUNTIMES%"
+set INSTALL_DIR=opt\datadog-agent
```

Comment:
Did it work for windows build? Is it a new directory to be created ? In what parent directory?

Reply:
It worked as this is only used for knowing the subdirectory in which to locate the cache, so it would be created in the value provided by `OMNIBUS_GIT_CACHE_DIR`, which points to `C:\TEMP\omnibus-git-cache` by default on Windows.
However, I believe this is actually not unneeded. I removed it and will check the pipeline results